docs/our_work/ds278_Assurance_Research_Path.md

```diff
@@ -0,0 +1,29 @@
+---
+title: 'DS_278: Assurance Research Path'
+summary: 'Investigating new techniques to test and assure AI and machine learning solutions in the NHS'
+origin: 'NHS England'
+tags: ['NLP','LLM','GENAI',Assurance, Machine Learning, Classification, Computer Vision, Work in Progress]
+---
+
+The NHS AI Quality Community of Practice (AIQ CoP) is a network of testing and assurance professionals dedicated to supporting the assurance of healthcare AI. While not an assurance function itself, the AIQ CoP provides guidance, resources, and advice to assist colleagues and organisations in navigating the complexities of AI assurance. While the project sits within NHS England, the remit of the AIQCOP is the whole of the NHS. 
+
+The AIQCOP conducts cutting-edge research to address real-world challenges in AI assurance. Current research areas include boundary analysis in image classifiers, focusing on understanding and validating how AI models make classification decisions; bias in AI training data, investigating scenario and demographic biases to mitigate risks in model performance; literacy bias in chatbots and AI search systems, aiming to ensure equitable access for users with varying literacy levels; and exploring AI-driven automation of risk logs, test scripts, and test data generation. This work combines traditional assurance techniques with novel approaches to enhance the reliability and fairness of AI systems.
+
+For years now the AIQCOP has relied on embedded Data Scientists from NHS England to help advise on the technical aspects of AI Assurance. 
+
+Read our Mixup Assurance paper here:  - [Link to paper](https://arxiv.org/abs/2311.13978)
```

Comment:
```suggestion
Read our Mixup Assurance paper here [on ArXiv](https://arxiv.org/abs/2311.13978)
```

just think it looks nicer this way! might also be worth giving a very brief explanation of what Mixup is